Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/Notifications/ToastWithTextBox.cpp

```diff
@@ -44,21 +44,16 @@ bool ToastWithTextBox::SendToast()
             </toast>" };
 
     auto toast{ winrt::AppNotification(xmlPayload) };
-    toast.Priority(winrt::AppNotificationPriority::High);
```

Comment:
nit: we don't need this?

Reply:
Most app notifications don't need this. High priority is only useful for a very specific Settings thing.